components/Navbar/Navbar.js

```diff
@@ -3,9 +3,15 @@ import Link from "next/link";
 const Navbar = () => {
   return (
     <nav className="navbar navbar-expand-lg  navbar-dark bg-dark ">
-      <div className="container-fluid">
+      <div className="container-fluid homebox">
         <a className="navbar-brand" href="#">
-          <Image src="/assets/images/logo.png" width={130} height={77} alt="" />{" "}
+          <img
+            src="https://pps.whatsapp.net/v/t61.24694-24/158318743_498252878509049_663621015264115443_n.jpg?ccb=11-4&oh=01_AVyllSGcdF9fqVxG50OVLNHY_49jamkcJdN0OBslszHErQ&oe=62D6BDB8"
```

Comment:
add image properly to assets folder

Reply:
It would be better if we call it from a CDN link, as it will make the page more light weight

Reply:
is the cdn link valid? this seems like a WhatsApp link 

<img width="1512" alt="Screenshot 2022-09-18 at 11 20 28 PM" src="https://user-images.githubusercontent.com/90271194/190921386-e46c7660-fa7a-4b76-9f88-6cd1b95fc9f3.png">